docs/troubleshooting/transferring-your-app.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Issues with transferring the ownership of your application
+description: A guide covering some of the edge cases when transferring your application
+---
+
+# Transferring ownership of your application
+To transfer your application, head to the [Settings view](https://dashboard.clerk.com/last-active?path=settings) within the Clerk Dashboard and select `Transfer Ownership`. From here you can transfer an application to your "Personal Account" or any Organization where you are an admin. 
+
+
+## Transferring to an Org that does not have any billing methods
```

Comment:
```suggestion
## Transfer to an org without payment methods
```

Based on [this error](https://docs-preview-837.clerkpreview.com/docs/errors/application#cannot-transfer-to-account-without-payment-method), I think we call it a payment method rather than a billing method, correct? 

If not, you can ignore that part, but we should drop the "ing" in "Transferring", and I think my simplification of the heading makes it take up less space in the ToC